fire-cakes/pancakes
# Model

What does this PR do?
I make some changes on games_controller. The board looks great and app is functioning in localhost

### Where should I start?
Merge and pull to get the latest changes

### How do I manually test this?
To check out Dan chess board, just update your master branch, rails s, games/show.

### Additional Comments
Rubocop: 59 files inspected, 40 offenses detected, 18 offenses corrected


### Trello story
[List or link]

### GIF for how this PR makes me feel
![](GIF_URL)
